.gitignore

```diff
@@ -3,4 +3,3 @@ elm-stuff
 dist
 assets
 *.DS_Store
-.vscode
```

Comment:
How much of the `.vscode` folder do we want to commit? For example, this will pick up a `tasks.json` file. Do we want to switch to committing the folder by default, but ignoring specific files?

Reply:
Good shout, maybe we just add `extensions.json` for now?

Reply:
Idm, not sure what best practice is here regarding the folder in general.

Reply:
Maybe for now?

```.gitignore
.vscode
!.vscode/extensions.jsons
```